modules/custom/cu_forms_bundle/cu_forms_bundle.install

```diff
@@ -120,3 +120,9 @@ function cu_forms_bundle_update_7009() {
   watchdog('cu_forms_bundle', t("Disabled progress bar for @count single-page webforms.", array('@count' => $updated_count)));
 }
 
+/**
+ * Implements hook_disable().
+ */
+function cu_digital_campaign_bundle_disable() {
```

Comment:
Function name is incorrect and causing a fatal error

Reply:
Oh, yes.